OpenBullet/MainWindow.xaml.cs

```diff
@@ -372,6 +372,33 @@ private void minimizePanel_MouseDown(object sender, MouseButtonEventArgs e)
             try { WindowState = WindowState.Minimized; } catch { }
         }
 
+        private void maximizePanel_MouseDoubleClick(object sender, MouseButtonEventArgs e)
+        {
+            if (e.ClickCount == 2)
+            {
+
+                if (maximized)
+                {
+                    Rect workArea = SystemParameters.WorkArea;
+                    this.Width = 800;
+                    this.Height = 620;
```

Comment:
Please use the values from settings (OpenBullet -> General -> Starting Window Width / Height) instead of resizing to the default size, so a user who specifies a different value can have it returned to that size he prefers!